app/build.gradle

```diff
@@ -192,21 +193,26 @@ dependencies {
 
     // Monitoring - Leak Canary
     debugImplementation "com.squareup.leakcanary:leakcanary-android:$leakcanaryVersion"
+    debugImplementation 'androidx.fragment:fragment-testing:1.1.0-alpha04'
```

Comment:
Do we still need this dependency? If we do, we probably want to make sure it is scoped to `testImplementation`.